Documentation/network/kubernetes/intro.rst

```diff
@@ -79,4 +84,3 @@ Networking Model
 <https://kubernetes.io/docs/concepts/cluster-administration/networking/>`_ and
 `Kubernetes Network Plugins
 <https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/>`_.
-
```

Comment:
Unrelated.

Reply:
This trailing space is actually in the file already, if you edit the file, you'd see its there. I'm not sure why Github isn't pick it up

Reply:
There's no trailing space here. This is a blank line that you remove in your PR.